molecule/scenario_resources/verify.yml

```diff
@@ -82,3 +82,11 @@
         msg: "Inspec failed to validate"
       when: item.rc != 0
       with_items: "{{ test_results.results }}"
+
+    - name: Check RAM usage
+      command: free -m
+      changed_when: false
+      register: result
+
+    - name: Show RAM usage, so we know if our CI env is low on RAM
+      debug: var=result.stdout_lines
```

Comment:
Cool! But I wonder if molecule has something for it

Reply:
For the amount of RAM, yes. For currently free, it doesn't seem like it. I researched it for a bit.